confluent_kafka_helpers/consumer.py

```diff
@@ -72,9 +74,10 @@ def __init__(
         self.config['error_cb'] = get_callback(config.pop('error_cb', None), default_error_cb)
         self.config['stats_cb'] = get_callback(config.pop('stats_cb', None), default_stats_cb)
         self.topics = self._get_topics(self.config)
+        self.logger = logger
 
-        logger.info("Initializing consumer", config=self.config)
-        self.consumer = ConfluentAvroConsumer(self.config)
+        self.logger.info("Initializing consumer", config=self.config)
+        self.consumer = ConfluentAvroConsumer(self.config, logger=self.logger, **kwargs)
```

Comment:
Do we want to always pass a logger instance or just not pass anything when no logger is passed to the constructor?